vendor/bat-native-ledger/src/bat/ledger/internal/properties/current_reconcile_properties.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_CURRENT_RECONCILE_PROPERTIES_H_
+#define BRAVELEDGER_CURRENT_RECONCILE_PROPERTIES_H_
+
+#include <stdint.h>
+#include <string>
+#include <map>
+
+#include "bat/ledger/internal/properties/surveyor_info_properties.h"
+#include "bat/ledger/internal/properties/reconcile_direction_properties.h"
+#include "bat/ledger/mojom_structs.h"
+
+namespace ledger {
+
+struct CurrentReconcileProperties {
+  CurrentReconcileProperties();
+  CurrentReconcileProperties(
+      const CurrentReconcileProperties& properties);
+  ~CurrentReconcileProperties();
+
+  bool operator==(
+      const CurrentReconcileProperties& rhs) const;
+
+  bool operator!=(
+      const CurrentReconcileProperties& rhs) const;
+
+  std::string viewing_id;
+  std::string anonize_viewing_id;
+  std::string registrar_vk;
+  std::string pre_flight;
+  std::string master_user_token;
+  SurveyorInfoProperties surveyor_info;
+  uint64_t timestamp;
+  std::map<std::string, double> rates;
+  std::string amount;
+  std::string currency;
+  double fee;
+  ReconcileDirections reconcile_directions;
```

Comment:
this could be just `directions` as reconcile is already in the name of struct

Reply:
Reconcile is in the name of the `struct` but the object here is `ReconcileDirections` should want to be clear or are you saying also rename "ReconcileDirections` to just `Directions`?

Reply:
we have in code now mostly something like this `reconcile.reconcile_directions`

Reply:
directions are only used for reconcile, so we could drop reconcile yes